index.html

```diff
@@ -25,11 +25,11 @@
   <meta name="description" content="Free software for typesetting Gregorian chant" lang="en">
   <meta name="keywords" content="Gregorian chant, gregorio, gabc, gregoriotex" lang="en">
   <meta name="author" content="The Gregorio project">
-  <link rel="shortcut icon" href="/illus/fav.ico">
-  <link rel="icon" href="/illus/fav.ico">
-  <link rel="stylesheet" href="/style.css" type="text/css" title="default">
+  <link rel="shortcut icon" href="./illus/fav.ico">
+  <link rel="icon" href="./illus/fav.ico">
+  <link rel="stylesheet" href="./style.css" type="text/css" title="default">
```

Comment:
If I recall correctly, I used the explicit `./` in order to be consistent with the `../` that appears on the pages which are buried in the folder structure of the site.  This consistency made it easier to relativize all the internal links as then sed or a "Find and Replace..." search could tell the difference between a link which had been fixed (and thus started with a `.` and one which had not.  I'm certainly willing to change it if people think that's a good idea, I just wanted to explain my reasoning for why I did what I did.